### PR TITLE
[Installer] Mark bundle install on upgrade to 1.1

### PR DESCRIPTION
**Pull request by dvesh3**
Resolves #435

### CODE REVIEW

File: src/Installer.php [hunk 1]
```diff
@@ -61,9 +62,29 @@ public function install()
...
 
+        parent::install();
+
         return true;
     }
 
+    public function isInstalled()
+    {
+        // We need to explicitly mark this bundle installed, if Settingstore entry doesn't exists and datahub permission is installed
+        // e.g. updating from 1.0.* to 1.1.*
```
Comment (dvesh3) [inline src/Installer.php]:
```suggestion
        // We need to explicitly mark this bundle installed, if Settingstore entry doesn't exists and datahub permission is installed
        // e.g. updating from 1.0.* to 1.1.*
```
```suggestion
        // When switching to SettingsStoreAwareInstaller, we need to explicitly mark this bundle installed, if Settingstore entry doesn't exists and datahub permission is installed
        // e.g. updating from 1.0.* to 1.1.*
```

**Comment by NiklasBr** [comment]
Got this error on installation of 1.1.1 @dvesh3 

```
[notice] Migrating up to Web2PrintToolsBundle\Migrations\PimcoreX\Version20210305134111
[warning] Migration Pimcore\Bundle\DataHubBundle\Migrations\PimcoreX\Version20210305134111 was executed but did not result in any SQL statements.
[error] Migration Pimcore\Bundle\DataHubBundle\Migrations\PimcoreX\Version20210305134111 failed during Post-Checks. Error: "An exception occurred while executing 'INSERT INTO migration_versions (`version`, `executed_at`, `execution_time`) VALUES (?, ?, ?)' with params ["Pimcore\\Bundle\\DataHubBundle\\Migrations\\PimcoreX\\Version20210305134111", "2021-11-12 15:04:07", 67]:

SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'Pimcore\Bundle\DataHubBundle\Migrations\PimcoreX\Version20210305' for key 'migration_versions.PRIMARY'"

In AbstractMySQLDriver.php line 74:

  An exception occurred while executing 'INSERT INTO migration_versions (`version`, `executed_at`, `execution_time`) VALUES (?, ?, ?)' wi
  th params ["Pimcore\\Bundle\\DataHubBundle\\Migrations\\PimcoreX\\Version20210305134111", "2021-11-12 15:04:07", 67]:

  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'Pimcore\Bundle\DataHubBundle\Migrations\PimcoreX\Version20210305
  ' for key 'migration_versions.PRIMARY'


In Exception.php line 18:

  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'Pimcore\Bundle\DataHubBundle\Migrations\PimcoreX\Version20210305
  ' for key 'migration_versions.PRIMARY'


In PDOStatement.php line 117:

  SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'Pimcore\Bundle\DataHubBundle\Migrations\PimcoreX\Version20210305
  ' for key 'migration_versions.PRIMARY'
```

**Comment by dvesh3** [comment]
> Got this error on installation of 1.1.1 @dvesh3

@NiklasBr thanks for reporting! fixed https://github.com/pimcore/data-hub/commit/a98b64591c12fa8f5e6dea96d6cb28f6c43626d7 and tagged new release v1.1.2

**Comment by NiklasBr** [comment]
I think this one worked 👍

**Pull request merged**